jupyterlab_sql_editor/ipython/sparkdf.py

```diff
@@ -128,3 +114,43 @@ def register_display():
     ip = get_ipython()
     plain_formatter = ip.display_formatter.formatters['text/plain']
     plain_formatter.for_type_by_name('pyspark.sql.dataframe', 'DataFrame', pyspark_dataframe_custom_formatter)
+
+# Code inspired from spark's dataframe.py
+def to_html(df, max_num_rows, truncate, show_nonprinting):
```

Comment:
Maybe worth adding a docstring to describe inputs/outputs and general purpose of function.